testdata/dnn/darknet/convolutional.cfg

```diff
@@ -0,0 +1,21 @@
+[net]
+batch=1
+channels=3
+height=4
+width=4
```

Comment:
Consider using different values for `weight`/`height`.

Reply:
/cc @ashishkrshrivastava

Reply:
@alalek @dkurt, sorry for such a late response. I have changed width value.

Reply:
@alalek , assuming you meant `width/height`